packages/simple-web-components/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@small-advertisements/simple-web-components",
```

Comment:
```suggestion
  "name": "@small-advertisements/ui",
```

Reply:
Done

Reply:
Can you also rename the whole file once again.

Reply:
I mean the file in the `packages` to match the name we gave here, aka `ui`

Reply:
I thought about it but I wasn't 100 % sure. Done

Reply:
@AdrianValdes check this ☝️